examples/multiclass/plot_multiclass_overview.py

```diff
@@ -0,0 +1,189 @@
+"""
+==================================
+Multiclass classification overview
```

Comment:
I have the sensation that one may expect to read about multiclass evaluation by reading this title. I suggest an alternative title similar to "Overview of multiclass training meta-estimators".

A possible alternative is to add a small paragraph such as:

One should not confuse the OvO/OvR strategy used for the **evaluation** of multiclass classifiers with the OvO/OvR strategy used to **train** a multiclass classifier by fitting a set of binary classifiers (the :class:`~sklearn.multiclass.OneVsOneClassifier` and :class:`~sklearn.multiclass.OneVsRestClassifier` meta-estimator). The OvR ROC evaluation can be used to scrutinize any kind of classification models irrespectively of how they were trained (see :ref:`multiclass`).

Maybe we can even link to the [Multiclass ROC example](https://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html).

Reply:
I will change the title and provide a portion of the proposed paragraph. Indeed, I don't want to go into details in the evaluation explanation but I want to point out the distinction.